src/renderer/components/screens/WelcomeScreen/chatmailInstances.ts

```diff
@@ -0,0 +1,22 @@
+import type { QrWithUrl } from '../../../backend/qr'
+
+// URL to list of various other chatmail instances
+export const CHATMAIL_INSTANCES_LIST_URL = 'https://delta.chat/chatmail'
```

Comment:
until we have translations ready (requires pushing to transifex, but that's too soon as the page is subject to change) better use the english page explicitly

```suggestion
export const CHATMAIL_INSTANCES_LIST_URL = 'https://delta.chat/en/chatmail'
```